### DISCUSSION TITLE
Using a variable as a fragment of a command

### DISCUSSION THREAD

**Discussion by xmnlab** [xonsh/xonsh#5049] [Q&A] [answered]
Hi everyone!

Using bash, I am able to create a variable and assign it, for example, the initial part of a command, and use that in the console and add to that more arguments, for example:

```bash
export MAKIM="makim --makim-file ./tests/.makim-simple.yaml"
${MAKIM} --help
```

I was able to do it with xonsh with:

```xonsh
@$(echo $MAKIM + ' --help')
```

is there a better (short) way to do that?

thanks!!

**Comment by gforsyth** [comment, marked as answer]
Hey @xmnlab !  Your syntax is correct, but you could make use of the `abbrevs` xontrib and do:

```xsh
xontrib load abbrevs
abbrevs["makim"] = "makim --makim-file ./tests/.makim-simple.yaml"
makim<space> --help
```

**Reply by xmnlab** [reply]
that looks very nice! thanks @gforsyth for your answer!! really appreciate that!
